manual_jb/content/contact.md

```diff
@@ -1,5 +1,9 @@
 # Contact Information
 
-* Contact [NCSA](http://www.ncsa.illinois.edu/) team via <incore-dev@lists.illinois.edu> regarding use of IN-CORE. The team will try to respond within 24 hours during business days.
+* You can contact the [NCSA](http://www.ncsa.illinois.edu/) team via <incore-dev@lists.illinois.edu> regarding
+  the use of IN-CORE. The team will try to respond within 24 hours during business days.
+* You can also reach out through our Slack channel. To get started, go to
+  [https://in-core.slack.com/](https://in-core.slack.com/). Or, you can grab an invitation here: [https://join.
+  slack.com/t/in-core/shared_invite/zt-25zffgnae-h0v8uGjpSli1YYp0Ypr68Q](https://join.slack.com/t/in-core/shared_invite/zt-25zffgnae-h0v8uGjpSli1YYp0Ypr68Q)
```

Comment:
I see an empyt space between "." and "slack" somehow 
<img width="159" alt="image" src="https://github.com/IN-CORE/incore-docs/assets/13950475/d8764fb6-c9b7-42bf-9e3b-fe1276c53370">

Reply:
The link works correctly so I can't tell if the space is actually there or is the browser is correcting the link. I'll move the link to a single long line so there's definitely no space.

Reply:
the link worked

Reply:
Fixed